src/main/java/com/games/online/shop/domain/Order.java

```diff
@@ -0,0 +1,131 @@
+package com.games.online.shop.domain;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "shop_order")
+public class Order extends AbstractAuditingEntity implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sequenceGenerator")
+    @SequenceGenerator(name = "sequenceGenerator")
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id", referencedColumnName = "id")
+    private User userId;
```

Comment:
I think this field should be called `user` because in the entity it will be the entire user and not just their id

Reply:
Good point - will change!